### PR TITLE
003 Script for requirement [PerformAudioPassThru] SDL must transfer reque…

### PR DESCRIPTION
**Pull request by maria-mih**
…st to HMI in case of valid audioPassThruIcon param
003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua 

All common functions used available at #703
Testing coverage available at #643
@istoimenova Please review

### CODE REVIEW

**Comment by istoimenova** [comment]
@maria-mih, please reference script in the TestSet.

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+-- as well as another related to request valid params
+-- SDL must transfer UI.PerformAudioPassThru (<audioPassThruIcon>, other params)_request + Speak_request (depends on parameters provided by the app) to HMI
+--1. Used preconditions
+--1. All params used in PerformAudioPassThru_request are present and in lower bound. 
+--2. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+--2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, other params) from mobile to SDL and check:
+--2.1 SDL sends UI.PerformAudioPassThru (audioPassThruIcon, other params) to HMI
+--2.2 SDL sends TTS.Speak to HMI
+--2.3 HMI sends UI.PerformAudioPassThru (SUCCESS) to SDL
```
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @istoimenova Updated in commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+--2.4 HMI sends TTS.Speak (SUCCESS) to SDL
+-- Expected result:
```
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @istoimenova Updated in commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable ()
```
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @istoimenova Updated in commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+commonSteps:PutFile("Precondition_PutFile_With_Icon","1")
+
+function Test:Precondition_Check_audioPassThruIcon_Existence()
+  testCasesForPerformAudioPassThru:Check_audioPassThruIcon_Existence(self)
+end
+
+function Test:Precondition_ActivateApp()
+  testCasesForPerformAudioPassThru:ActivateAppDiffPolicyFlag
+  (self, config.application1.registerAppInterfaceParams.appName, config.deviceMAC)
```
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @istoimenova Updated in commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
```
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @istoimenova Updated in commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+function Test:TestStep_PerformAudioPassThru_AllParameters_Lower_SUCCESS()
+  testCasesForPerformAudioPassThru:PerformAudioPassThru_AllParameters_Lower_SUCCESS (self)
```
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @istoimenova Updated in commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

**Comment by maria-mih** [comment]
@istoimenova  Script has been updated after first review iteration. Please check on commit 4d5b7b28a1aa845ec5d45eed99d4e92d05eba3e5

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @maria-mih, ACCEPTED.

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @maria-mih, ACCEPTED.

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @maria-mih, ACCEPTED.

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua (hunk 4 above)
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @maria-mih, ACCEPTED.

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua (hunk 5 above)
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @maria-mih, ACCEPTED.

File: test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua (hunk 6 above)
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/003_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Lower.lua]: @maria-mih, ACCEPTED.

**Pull request merged**